tests/src/init.rs

```diff
@@ -43,6 +43,10 @@ pub fn initialize_instance(backends: wgpu::Backends, force_fxc: bool) -> Instanc
                 shader_compiler: dx12_shader_compiler,
             },
             gl: wgpu::GlBackendOptions::from_env_or_default(),
+            // TODO: Use noop backend in tests instead of leaving it disabled by default?
```

Comment:
Could this be an issue? Todos aren't terribly visible

Reply:
https://github.com/gfx-rs/wgpu/issues/7119